src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/Utilities.cs

```diff
@@ -15,7 +16,14 @@ namespace Microsoft.DotNet.SourceBuild.SmokeTests;
 
 public static class Utilities
 {
-    public static void ExtractFileFromTarball(string tarballPath, string filePath, string outputDir)
+    public static void ExtractTarball(string tarballPath, string outputDir)
+    {
+        using FileStream fileStream = File.OpenRead(tarballPath);
+        using GZipStream decompressorStream = new(fileStream, CompressionMode.Decompress);
+        TarFile.ExtractToDirectory(decompressorStream, outputDir, true);
+    }
+
+    public static void ExtractTarball(string tarballPath, string outputDir, string filePath)
```

Comment:
Since the method name's been changed, consider renaming `filePath` to `targetFilePath` to make it more clear what it is for.

Reply:
Done